bspump/kafka/source.py

```diff
@@ -108,6 +110,14 @@ def __init__(self, app, pipeline, connection, id=None, config=None):
 
 		# Sleep time after an error
 		self.SleepOnError = float(self.Config.pop("sleep_on_error", 3.0))
+		self.SleepOnRetryableError = float(self.Config.pop("sleep_on_retryable_error", 60.0))
+
+		# Retryable errors retry attempts
+		self.ErrorsMaxRetryAttemps = int(self.Config.pop("errors_max_retry_attempts", 100))
```

Comment:
With this set, you get the first warning after 6000 seconds? Shouldn't you wait for example 1 second and display the warning after 1 minute?

Reply:
I don't think the warning is relevant to the user honestly.

Reply:
Let us see if there are more retryable errors. If I don't recreate the consumer, the error is seen only once either way.